src/components/layout/ViewContainer.js

```diff
@@ -5,6 +5,6 @@ const ViewContainer = styled.div`
   width: 100vw;
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+  justify-content: ${({top}) => top ? 'flex-start' : 'space-between'};
 `
```

Comment:
Avoid using logic inside the css

Reply:
Use 2 different components for each option! This will increase readability in the code for developers

Reply:
Done!